apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -102,7 +102,7 @@ export interface Location {
     /**
      * If the location only applies on specific days, this is non-null.
      */
-    days?: string[];
+    days: string | null;
```

Comment:
```suggestion
    days?: string;
```
I'm under the impression that this is artificially nullable; I'm pretty sure it should be safe to make it optional, i.e. it exists or it's undefined. The only way it would be null is if we ever set it as null ourselves (which I noticed below, which we shouldn't do if we want to keep the interface simpler)